### DISCUSSION TITLE
How to turn the "|" indent character off ?

### DISCUSSION THREAD

**Discussion by mlauronen** [LazyVim/LazyVim#1784] [Q&A] [answered]
I understand that this is a question of indent-blankline.nvim plugin but cannot figure out the correct place to do it.

**Comment by dpetka2001** [comment]
I would guess that you could do it by looking at the Lazyvim [docs](https://www.lazyvim.org/plugins/ui#indent-blanklinenvim) how it's configured. So, what i would try is to set these
```lua
indent = {
    char = "│",
    tab_char = "│",
  },
```
to just empty strings or whatever character you would like to use instead.

**Reply by mlauronen** [reply, marked as answer]
I disabled the whole plugin by [these instructions](https://www.lazyvim.org/configuration/plugins#-disabling-plugins) and now those characters are gone. Thank you.
